pandas/io/formats/style.py

```diff
@@ -545,7 +555,7 @@ def render(self, **kwargs):
         d.update(kwargs)
         return self.template.render(**d)
 
-    def _update_ctx(self, attrs):
+    def _update_ctx(self, attrs: FrameOrSeries) -> None:
```

Comment:
Does `attrs` here not have to be a DataFrame? Looks like it calls iterrows in the implementation which isn't defined for Series

Reply:
Then the docstring is not correct.
I fixed both the type annotation and the docstring in 6a06e50